KarhooUISDK/Screens/PassengerDetails/PassengerDetailsViewController.swift

```diff
@@ -39,6 +39,25 @@ final class PassengerDetailsViewController: UIViewController, BaseViewController
     private var shouldMoveToNextInputViewOnReturn = true
     private let currentLocale = NSLocale.current.languageCode ?? "en"
     
+    // MARK: - PassengerDetailsView
+    var delegate: PassengerDetailsDelegate? {
```

Comment:
Strong reference cycle

Reply:
Fixed in the `PassengerDetailsPresenter`